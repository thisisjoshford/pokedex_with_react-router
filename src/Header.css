.headerLogo {
    text-align: center;
    margin-top: 0px;
    margin-bottom: 0px;
}

.mainLogo {
    width:400px;
}

.classLabel {
    margin-bottom: 0px;
}