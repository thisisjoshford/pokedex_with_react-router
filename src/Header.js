import React, { Component } from "react";
import './Header.css'
export default class Header extends Component {

    render() {
        return(
            <div className="headerLogo">
                <img className="mainLogo" src="mainLogo.png" alt="pokemon"/>
            </div>
        );
    }
}