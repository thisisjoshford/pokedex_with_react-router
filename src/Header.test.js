import React from 'react';
import Header from './Header.js';
import renderer from 'react-test-renderer';

it('header renders correctly', () => {
  const tree = renderer
    .create(<Header/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});