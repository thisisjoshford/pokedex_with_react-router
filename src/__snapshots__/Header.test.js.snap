// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header renders correctly 1`] = `
<div
  className="headerLogo"
>
  <img
    alt="pokemon"
    className="mainLogo"
    src="mainLogo.png"
  />
</div>
`;
